packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -198,7 +236,7 @@ const styles = StyleSheet.create({
         },
     },
     error: {
-        background: "rgba(217, 41, 22, 0.06)",
+        background: `linear-gradient(0deg, rgba(217, 41, 22, 0.06), rgba(217, 41, 22, 0.06)), ${Color.white}`,
```

Comment:
I returned the `linear-gradient` because it gave the TextField a white background with the red hue. The old version would only give the red hue. I now understand why Nefaur had the linear gradient on the Figma like this.

Reply:
See comment [above](https://github.com/Khan/wonder-blocks/pull/1132#discussion_r644277785)